src/Form.jsx

```diff
@@ -103,6 +103,12 @@ class Form extends React.Component {
       checkValidation: () => this.checkValidation()
     };
 
+    if((field.type == 'text' || field.type == 'textarea') && !field.multiplicity) {
+      common.charLimit = field.charLimit
+      common.wordLimit = field.wordLimit
+      common.cutAtLimit = field.cutAtLimit
+    }
+
```

Comment:
I have by passed the counter creation for multiplicity fields for now.. The problem is this i cannot wrap the add counter on a multiplicity field but on individual fields in multiplicity field

Reply:
I am not sure what you mean with this, can you explain this a bit further?

Reply:
Well, if field is a multiplicity field, My HOC cannot handle that since it expects to be wrapped around a single text type field.

Reply:
So it would be better to wrap individual text fields inside a multiplicity field, if we can create an HOC for multiplicity field too. Then we can do something like this:

```
@addMultiple
@addCounter
class TextInput extends ReactComponent {}
```
```